bindings/C/c/adios2_c_attribute.cpp

```diff
@@ -31,9 +31,7 @@ adios2_error adios2_attribute_name(char *name, size_t *size,
         const adios2::core::AttributeBase *attributeBase =
             reinterpret_cast<const adios2::core::AttributeBase *>(attribute);
 
-        *size = attributeBase->m_Name.size();
-        attributeBase->m_Name.copy(name, *size);
-        return adios2_error_none;
+        return String2CAPI(attributeBase->m_Name, name, size);
```

Comment:
why not making this part of the helper namespace that returns a void? A more reusable name would be StringToChar, it doesn't have to return an error from enum that is exclusive to the C API. This function is a lot more generic and can be reused in other contexts, please move it to: https://github.com/ornladios/ADIOS2/blob/master/source/adios2/helper/adiosString.h (and in adiosString.cpp)

Reply:
I thought about helper:: when I wrote this, but I didn't want to go there, because I don't see any use outside of the C API for this function. Which is also why I named it that way and included
the error code return.
I wrote this tiny helper, but I don't like it, and I actually don't want it to be used anywhere else.

Reply:
that's how all functions in helper started: only one use, over time it helps us maintain code since we have one namespace or reusable generic components. We already have StringToDouble, DimsToString, etc.

Reply:
> I actually don't want it to be used anywhere else

Why not? We, the team, would prefer it under helper:: as that's how we've been operating for a while for reusability.